### ISSUE TITLE
Content configuration: glob pattern matching does not match paths containing parentheses

### ISSUE THREAD

**Issue by pm0u** [tailwindlabs/tailwindcss#9741]
**What version of Tailwind CSS are you using?**

3.2.1

**What build tool (or framework if it abstracts the build tool) are you using?**

NextJS 13

**What version of Node.js are you using?**

v16.18.0

**What browser are you using?**

N/A

**What operating system are you using?**

MacOS

**Reproduction URL**

https://github.com/pm0u/next-13-tailwind-parentheses

**Describe your issue**

Using parentheses in globbing patterns seems to cause tailwind to match no files in the directory. This is described by fast glob as requiring a double `\\` to escape, such as `./app/\\(route-group\\)/**/*.js` however this seems to generate no classes when used in this way. Since Next 13 supports global CSS imports anywhere as well as route grouping using parentheses in folder names, this should be a supported feature.

(I believe) it is possible to confirm that the parentheses are the issue by removing that part of the path in the example repo to just have one of the configs apply to all files in `./app/**/*.{js,jsx,ts,tsx}`

**Comment by thecrypticace** [comment]
Hey, we released v3.2.2 on Friday which fixes buggy behavior around paths with parens in them (because parens are special characters in globs). You _do not_ need to escape them yourself in your content globs — we do that for you.

The gist of what was happening is that we perform path normalization, even on content paths that are globs, which changes `\` into `/` because of Windows having a different path separator. This unfortunately causes your own attempt at escaping to break down and get converted into forward slashes.

But, as I mentioned, we'll do that for you so once you upgrade to v3.2.2 and update your paths to these:
```js
"./app/(layout-1)/**/*.{js,jsx,ts,tsx}"
"./app/(layout-2)/**/*.{js,jsx,ts,tsx}"
```

it'll work as you expect:
<img width="311" alt="Screen Shot 2022-11-07 at 14 58 41" src="https://user-images.githubusercontent.com/614993/200403041-3e0ad120-18bd-4912-be15-661a81c2c0f1.png">

**Comment by pm0u** [comment]
Thank you for the fast resolution and followup!!
